runtime/js/40_process.js

```diff
@@ -30,6 +30,7 @@ import {
   ReadableStreamPrototype,
   writableStreamForRid,
 } from "ext:deno_web/06_streams.js";
+import { setPipeFdCallback } from "ext:deno_node/internal/child_process.ts";
```

Comment:
Not sure why snapshot fails: https://github.com/denoland/deno/actions/runs/7204945647/job/19627152677?pr=21570#step:30:137

Any ideas?

Reply:
It's probably a circular dependency between `ext:deno_node` and `ext:runtime` that V8 can't handle. Try using `internals` namespace to store this function.